.github/workflows/ci.yml

```diff
@@ -17,6 +17,12 @@ jobs:
         secrets:
             HARDHAT_FORKING_URL: ${{ secrets.HARDHAT_FORKING_URL }}
 
+    foundry-tests:
```

Comment:
Please add the name of this step to `needs` section in `report` step.
Also to the `success` condition